autopush/web/registration.py

```diff
@@ -71,7 +74,19 @@ def convert_chid(self, data):
 class TokenSchema(SubInfoSchema):
     """Filters allowed values from body data"""
     token = fields.Str(allow_none=True)
-    # Temporarily allow 'aps' definition data for iOS.
+
+
+def validate_token(val):
```

Comment:
Why not use the http://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.validate.Length validator to ensure there's something here?